packages/cardano-services/test/Asset/DbSyncAssetProvider.test.ts

```diff
@@ -97,13 +99,30 @@ describe('DbSyncAssetProvider', () => {
     const history = await fixtureBuilder.getHistory(assets[0].policyId, assets[0].name);
 
     expect(asset.history).toEqual(history);
+    // TODO: review test data, this is false positive right now
+    // expect(asset.nftMetadata).toBeTruthy();
     expect(asset.nftMetadata).toStrictEqual(assets[0].metadata);
     expect(asset.tokenMetadata).toStrictEqual({
       assetId: '728847c7898b06f180de05c80b37d38bf77a9ea22bd1e222b8014d964e46542d66696c6573',
       desc: 'This is my second NFT',
       name: 'Bored Ape'
     });
   });
+  it.todo('caches asset info query responses');
```

Comment:
Todo

Reply:
This one might be difficult to test, either:
- need to refactor the code with a new interface/service that we can mock
- could assert # of queries made with the `Pool`, but I think that'd be a bad/brittle test that's too coupled to the implementation